light.py

```diff
@@ -0,0 +1,12 @@
+import neopixel
+
+class Light:
+
+    def __init__(self, channel, pixel_pin, pixel_count) -> None:
+        self.neopixel = neopixel.NeoPixel(pixel_pin, pixel_count, brightness=1, auto_write=False)
```

Comment:
⭐ ⭐ ⭐ I don't think you want to create a new NeoPixel object for each individual pixel, there should just be 1 per strip (pixel_pin)

Reply:
The intention was to create a new NeoPixel object for each strip of light.

Reply:
Ah great. I think I was confused about what 'channel' was in that case, my initial guess was the current pixel index. Could you elaborate on the intention behind that member?

Reply:
Also consider renaming the class `LightStrip` to be more explicit

Reply:
Channel is the "key" to access the RobotStates list. It's kind of like the motor ID we have on our main RoboRIO project where the code communicates with the motor, in this case, it is the key to access the information sent by the RoboRIO.